node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/docker/DockerNetworking.java

```diff
@@ -28,7 +28,7 @@ public String getDockerNetworkMode() {
     }
 
     public static DockerNetworking from(String cloud, NodeType nodeType, boolean hostAdmin) {
-        if (cloud.equals("AWS")) {
+        if (cloud.equals("aws")) {
```

Comment:
Use `.equalsIgnoreCase()` to be on the safe side?

Reply:
Agree